packaging/pom.xml

```diff
@@ -173,4 +173,8 @@
             </plugin>
         </plugins>
     </build>
+
+  <scm>
+    <tag>brooklyn-dist-0.9.0</tag>
```

Comment:
Should this be shared for the whole dist? Or is it even needed, haven't seen one used elsewhere in the poms.
Again if it's here to stay mark with <!-- BROOKLYN_VERSION -->, and change the version to 0.9.0-SNAPSHOT so that the marker actually works.